### DISCUSSION TITLE
Ability to add commits to git-blame-ignore-revs

### DISCUSSION THREAD

**Discussion by not7cd** [renovatebot/renovate#27590] [Suggest an Idea]
### Tell us more.

It is often desirable to ignore commits that only format code in `git blame`. Especially if these changes are made by bots. 
Renovate should allow configuration for adding commits to `.git-blame-ignore-revs` file, either for whole PR or `postUpgradeCmd` part. But because Renovate doesn't know the commit hash before squashing it, additional commit must be pushed to main branch.

https://www.stefanjudis.com/today-i-learned/how-to-exclude-commits-from-git-blame/

**Comment by rarkins** [comment]
Normal Renovate commits would seem like they should be part of git blame, because they are changing dependency versions and may be the source of problems. Which type of commit would you expect to be useful for ignoring?

**Reply by not7cd** [reply]
This config should give an example:

```
    {
      "description": "Run black after upgrade",
      "matchDatasources": ["pypi"],
      "matchPackageNames": ["black"],
      "postUpgradeTasks": {
        "commands": ["pip install black=={{{newVersion}}}", "black ."],
        "fileFilters": ["**/*.py"],
        "executionMode": "branch"
      },
    }
      
```

Changes created by commands in `postUpgradeTasks` should be in a single commit and then ignored.

**Reply by rarkins** [reply]
Changes in postUpgradeTasks are already combined with Renovate's single commit. But I disagree that they should be ignored, because they can be the cause of breakages
